src/sidebar/services/test/groups-test.js

```diff
@@ -230,6 +230,25 @@ describe('groups', function() {
       });
     });
 
+    // TODO: Add a de-dup test for the direct-linked annotation.
```

Comment:
I'm not sure which PR will get merged first (this one or https://github.com/hypothesis/client/pull/1097). Since this one is simpler I'm guessing this one.